examples/ensemble/plot_gradient_boosting_cv_vs_grid_search.py

```diff
@@ -0,0 +1,77 @@
+"""
```

Comment:
I'm not convinced this example is worth having. A nice benchmark, but users don't gain from playing with it; as much can be said ("Will always improve performance over GridSearchCV for searching over n_estimators") in narrative docs and what's new.